### PR TITLE
add zh/en financial-aid page

### PR DESCRIPTION
**Pull request by peihsuan**
add bothr zh and en financial-aid page

### CODE REVIEW

**Comment by peihsuan** [comment]
@Lee-W

File: src/templates/pycontw-2020/contents/zh/registration/financial-aid.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+{% block meta_description %}如何申請 PyCon Taiwan 財務補助，讓我們協助你參與會議。{% endblock meta_description %}
+
+
+{% block content %}
+
+<h1>財務補助</h1>
+
+<h2>方案介紹</h2>
+
+<p><a href="http://jessenoller.com/blog/2011/05/25/pycon-everybody-pays" target="_blank" rel="noopener">Everybody Pays（公平付費）</a>是 PyCon 的核心價值，亦即除特邀來賓外，所有的會議參與者，包括講者與志工，皆需自己付費，方能參加會議。在維持 Everybody Pays 此一核心價值的同時，我們也規劃了財務補助方案，希望協助對會議有興趣，但由於成本而難以參加的朋友們，能夠參與這次會議。我們採取審核制，請有需求的朋友主動與我們聯絡，我們收到所有申請後，會根據預算、申請者狀況等因素，決定是否給予補助，以及其資源多寡。</p>
```
Comment (Lee-W) [inline src/templates/pycontw-2020/contents/zh/registration/financial-aid.html]: We no longer use "Everybody pays". Instead, we use "Everybody contributes". See [PyCon TW 2020 CFP](https://tw.pycon.org/2020/zh-hant/speaking/cfp/).

File: src/templates/pycontw-2020/contents/en/registration/financial-aid.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+{% block meta_description %}Overview on how to submit a financial aid application to PyCon Taiwan, and allow us to help you attend the conference.{% endblock meta_description %}
+
+
+{% block content %}
+
+<h1>Financial Aid</h1>
+
+<h2>The Program</h2>
+
+<p><a href="http://jessenoller.com/blog/2011/05/25/pycon-everybody-pays" target="_blank" rel="noopener">Everybody Pays</a> is one of PyCon’s core principles. All attendees including speakers and volunteers have to pay for registering. While following this convention, PyCon Taiwan also has a financial aid program to help friends in the community, financially or otherwise, so more people are able join us in the big event.</p>
```
Comment (Lee-W) [inline src/templates/pycontw-2020/contents/en/registration/financial-aid.html]: We no longer use "Everybody pays". Instead, we use "Everybody contributes". See PyCon TW 2020 CFP.

**Comment by tai271828** [comment]
Neat. I am so looking forward to seeing this page lands!

**Pull request closed without merging**